tests/temp/en/users.php

```diff
@@ -0,0 +1,7 @@
+<?php
```

Comment:
We don't create temp files manually, we create it via code as in the other tests.